xcodeproj/internal/BUILD

```diff
@@ -3,6 +3,31 @@ load(
     "bool_setting",
     "string_setting",
 )
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
```

Comment:
Can you move this above the other load statement to keep it alphabetical (root directory versus a subdirectory)?

Reply:
Done